pom.xml

```diff
@@ -243,6 +244,13 @@
     <build>
         <defaultGoal>install</defaultGoal>
 
+        <testResources>
+            <testResource>
+                <filtering>true</filtering>
+                <directory>${basedir}/src/test/projects</directory>
```

Comment:
In that case we need to review the approach in order to not add the projects directory as a test resource.